### PR TITLE
Feature/#55 Create and delete custom lists integration

### PR DESCRIPTION
**Pull request by revodark**

### CODE REVIEW

File: frontend/src/components/CustomForm/FormBody.jsx [hunk 1]
```diff
@@ -58,6 +59,18 @@ const FormBody = ({
           )}
         </>
       )}
+      {onCancel && (
```
Comment (Rolly007) [inline frontend/src/components/CustomForm/FormBody.jsx:62]: now we have 3 buttons maybe we could take the buttons out of this component and render them into another dumb component

File: frontend/src/components/PaginationCard/PaginationCard.jsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+import PropTypes from 'prop-types';
+
+import {
+  Card,
+  CardActionArea,
+  CardMedia,
+  Typography,
+  IconButton
+} from '@material-ui/core';
+import HighlightOffIcon from '@material-ui/icons/HighlightOff';
```
Comment (Rolly007) [inline frontend/src/components/PaginationCard/PaginationCard.jsx:12]: Please, could you import this icon in the "utils/icons/index.js" file, and use it from there

File: frontend/src/components/CustomForm/FormBody.jsx (hunk 1 above)
Comment (revodark) [reply frontend/src/components/CustomForm/FormBody.jsx]: You're right!

File: frontend/src/components/PaginationCard/PaginationCard.jsx (hunk 2 above)
Comment (revodark) [reply frontend/src/components/PaginationCard/PaginationCard.jsx]: Thanks!

**Pull request merged**